data/json/monsters/mammal.json

```
@@ -1759,6 +1759,83 @@
     },
     "flags": [ "ANIMAL", "DOGFOOD", "CANPLAY", "HEARS", "KEENNOSE", "PATH_AVOID_DANGER_1", "SEES", "SMELLS", "WARM" ]
   },
+  {
+    "id": "mon_dog_gswiss",
+    "copy-from": "mon_dog",
+    "type": "MONSTER",
+    "name": { "str_sp": "Greater Swiss" },
+    "description": "The biggest of four breeds of Swiss mountain dogs.  The Greater Swiss mountain dog was originally meant to pull carts, herd cattle and guard the homestead.  This one seems feral, but perhaps you could try taming it?",
```

```suggestion
    "description": "A feral Greater Swiss, biggest of four breeds of Swiss mountain dogs originally meant to pull carts, herd cattle and guard the homestead.",
```
A bit too wordy and prescriptive.